src/legacy/ui/public/vis/editors/default/_sidebar.scss

```diff
@@ -225,3 +225,7 @@
   margin-top: $euiSizeS;
   margin-bottom: $euiSizeS;
 }
+
+.visEditorSidebar__switchControl label{
+  display: initial;
```

Comment:
What is this intended to do? `.euiSwitch__label`'s don't have a `display` attribute applied.

Reply:
Unfortunately all labels inside the `visEditorSidebar` have the `display: block` option

![image](https://user-images.githubusercontent.com/31325372/55565243-28594180-5702-11e9-9a95-9c40fb12405f.png)

Reply:
I suppose we will avoid these rules when getting rid of legacy components

Reply:
Ugh! This is why we try to avoid generic/element selectors. I've got a hack we can apply to make sure these styles don't apply to EUI components. Can you change that original label selector to be:

`.visEditorSidebar__section label:not([class^="eui"])`

Reply:
@cchaos thanks for the hack 😉